unity-client/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/CatalogController.cs

```diff
@@ -35,11 +37,41 @@ public static WearableDictionary wearableCatalog
         }
     }
 
+    private static SceneAssetPackDictionary sceneAssetPackCatalogValue;
+    public static SceneAssetPackDictionary sceneObjectCatalog
+    {
+        get
+        {
+            if (sceneAssetPackCatalogValue == null)
+            {
+                sceneAssetPackCatalogValue = Resources.Load<SceneAssetPackDictionary>("SceneAssetPackCatalog");
```

Comment:
Could we create a constant for "SceneAssetPackCatalog"?

Reply:
In the same controller, there are more hardcode strings like "WereableCatalog", and "SceneAssetPackCatalog" will you like add those as constant too?

Reply:
Yes, I think it is better to group all these kind of hardcoded strings as constants in the same place of the script in order to make them easy to find and change (if needed).